.rubocop.yml

```diff
@@ -0,0 +1,31 @@
+# Rocketships!!!!
+HashSyntax:
+  EnforcedStyle: hash_rockets
```

Comment:
I'm going to pull out my CTO hat here. We've had this discussions many times and we even made a decision once.

Hashrockets it is